AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -156,14 +162,21 @@ - (void)setChildren:(NSArray *)children
 
 - (id<ASLayoutable>)child
 {
-  return _child;
+  return [_children firstObject];
```

Comment:
`firstObject` won't crash. It will return nil.
https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSArray_Class/#//apple_ref/occ/instp/NSArray/firstObject